test/routes_helpers.js

```diff
@@ -14,3 +23,49 @@ exports.getRoute = function (routes, path) {
 
   return route
 }
+
+exports.makeAccountRoutes = function (options, requireMocks) {
```

Comment:
> each of the new route modules will need their own makeRoutes implementation

Yea that is true, could possibly make it take a path to help reduce complexity?

Refactoring the `lib/routes/account.js` did cross my mind, but decided against it because it was a much bigger refactor than this. My goal here was to make testing easier and more manageable. Navigating around the `test/account_tests.js` was becoming a challenge.